pype/tools/tray/__main__.py

```diff
@@ -1,6 +1,7 @@
 import os
 import sys
-import pype_tray
+
+from . import pype_tray
```

Comment:
I think it is not allowed to do relative import in `__main__`

Reply:
sure, but settings and tray are now launched like modules like `python -m pype.tools.tray`

Reply:
In that case remove `__main__.py` files please.